api/models/node.go

```diff
@@ -37,11 +42,28 @@ type IndexFind struct {
 	From     int64    `json:"from" form:"from"`
 }
 
+func (i IndexFind) Trace(span opentracing.Span) {
+	span.SetTag("q", i.Patterns)
+	span.SetTag("org", i.OrgId)
+	span.SetTag("from", i.From)
+}
+
 type GetData struct {
 	Requests []Req `json:"requests" binding:"Required"`
 }
 
+func (g GetData) Trace(span opentracing.Span) {
+	for _, r := range g.Requests {
+		r.TraceLog(span)
+	}
+}
+
```

Comment:
i suspect this may be overdoing it. for every request in a GetData RPC call, log all the request properties. this could be too much overhead